build/centos7-puppet6/Puppetfile

```diff
@@ -55,7 +57,12 @@ mod 'puppet-nginx'
 mod 'puppetlabs-concat'    # dependency of puppet-nginx'
 mod 'puppet-nodejs'
 mod 'puppet-python'
-mod 'puppet-rabbitmq'
+# Waiting on https://github.com/voxpupuli/puppet-rabbitmq/pull/907 to be
+# merged into upstream repo.
+mod 'puppet/rabbitmq',
+    :git => 'https://github.com/EncoreTechnologies/puppet-rabbitmq.git',
+    :branch => 'feature/fix-yum-repo',
+    :default_branch => 'master'
```

Comment:
As https://github.com/voxpupuli/puppet-rabbitmq/pull/907 was merged,
looks like we can rely on the official version now :+1: